packages/grafana-ui/src/components/Combobox/getComboboxStyles.ts

```diff
@@ -78,7 +95,8 @@ export const getComboboxStyles = (theme: GrafanaTheme2) => {
     }),
     optionFocused: css({
       label: 'combobox-option-focused',
-      top: 0,
+      // Commenting this line avoids the option to move to the top and place under the group label
```

Comment:
@tskarhed, if I uncomment the `top: 0` when on hover the options move 🤦‍♀️  I don't have a clear idea of how to fix it, tbh